elastic/integrations
# [Enhancement] [integration] Add EclecticIQ Advanced CSV Outgoing feed integration

## Proposed commit message

Add Eclectic IQ integration with Advanced CSV Observables Outgoing Feed data stream.

The integration's data stream downloads outgoing feed content blocks data in CSV format, parses the CSV data and submits it to the ingestion pipeline.

Ingestion pipeline takes the input data and populates appropriate ECS fields.

The integration has a transform that creates "latest" index and populates it with records from data stream index that doesn't have "deleted_at" date, which is populated only for input records that are deleted from source dataset. This way, result index has only the latest data and provides a retention policy.

## Checklist


## Author's Checklist

- [ ] Review documentation

## How to test this PR locally

Prerequiseites
* Advanced CSV Observables Outgoing Feed with populated dataset(s).
* EIQ IC API key.

Steps:
* Install the integration.
* Set up the integration (EIQ IC URL, API token, outgoing feed ID).
* Check "logs-ti_eclecticiq_latest.observables-*" index for data.
